connector.yml

```diff
@@ -0,0 +1,24 @@
+---
+version: '3'
+
+services:
+  transmart-variant-store-connector:
+    container_name: transmart-variant-store-connector
+    image: thehyve/transmart-variant-store-connector
```

Comment:
Maybe add version `:0.0.1`?

Reply:
Ok. I'll do that. I was not sure if it is required.